src/messages/predicates/AuthorPredicate.cpp

```diff
@@ -0,0 +1,18 @@
+#include "messages/predicates/AuthorPredicate.hpp"
+
+namespace chatterino {
+
+AuthorPredicate::AuthorPredicate(const QStringList &authors)
+    : authors_(authors)
+{
+}
+
+bool AuthorPredicate::appliesTo(const MessagePtr message)
+{
+    if (!message)
```

Comment:
This check can be avoided by taking a `Message&` instead of a `MessagePtr` as a parameter instead.

Reply:
Resolved in 84be38f8.